### DISCUSSION TITLE
Can't assign vrrp priority per vlan?

### DISCUSSION THREAD

**Discussion by ddutt** [ipspace/netlab#1538] [Q&A]
In a classic acc-agg architecture you typically have 2 agg switches, one of which is STP root for one set of VLANs, and the other is root for the other set of VLANs. The ones which are root for a VLAN are also the primary vrrp gateway for that VLAN. The way to achieve that would be via the node.vlan.vrrp.priority attribute I thought. But I cannot specify priority per vlan on a node. So, how do I achieve what I want?

**Comment by jbemmel** [comment]
@ipspace another candidate use case for node level vlan groups.

Issue is closed but for completeness:
‘’’
nodes:
 agg1:
  vlans:
   v1:
     vrrp.priority: 200
‘’’
